### ISSUE TITLE
Allow line-disabling of eslint rules w/ no-inline-comments on

### ISSUE THREAD

**Issue by wbyoung** [eslint/eslint#3472]
When you enable no-inline-comments, you can't use inline comments to disable other eslint rules.

It'd be nice to be able to do this even with `no-inline-comments` enabled:

```js
var my_variable = 0; // eslint-disable-line camelcase
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by sindresorhus** [comment]
:+1: I've been hit by this too. I think it would make sense to make an exception for ESLint directives. The alternative is having to define a ESLint directive to be able to define another directive or using the open/close ESLint directive, but that looks very ugly and verbose.

**Comment by InsomniacFury** [comment]
Ideally this would also incorporate the ignoring of max-len.
For example right now, I'd have a line as verbose as:
```
 // eslint-disable-line max-nested-callbacks, no-inline-comments, max-len
```
